pages/index.tsx

```diff
@@ -102,50 +102,42 @@ const dotContent: Record<DotKey, DotContent> = {
   [DotKey.hasTokens]: {
     id: DotKey.hasTokens,
     position: 1,
-    toolTip: "Get 100 $FWEB3 tokens",
-    link: "https://fweb3.notion.site/Walkthrough-8ac4fc0d3b814a068767c86d63fd8fb7#4f53243dd1cd45008eb9152a881aa360",
```

Comment:
Don't think these links are actually used anywhere anymore since we now have this stuff inline.